config/defaults.conf

```diff
@@ -79,13 +79,13 @@ allow_reuse_of_pretraining_parameters = 0  // Set to 1 to allow task models that
                                            // reused in do_target_task_training. This may behave incorrectly if a run is stopped and
                                            // restarted in do_target_task_training (issues #285, #290).
 allow_missing_task_map = 0  // Dangerous: If true, ignore missing classifier_task_map.json
-                            // This is needed for bare-ELMo probing, since the main training phase is skipped for these models.
+                            // This is needed for bare-ELMo probing, since the pretraining training phase is skipped for these models.
 reload_tasks = 0     // If true, force the rebuilding of the task files in the experiment directory, even if they exist.
 reload_indexing = 0  // If true, force the rebuilding of the index files in preproc/ for tasks in reindex_tasks, even if they
                      // exist.
-eval_data_fraction = 1  // Use only the specified fraction of the training data in the do_target_task_training phase.
-                        // should not impact main training, even for the same task.
-                        // training_data_fraction should not be used together with eval_data_fraction
+target_train_data_fraction = 1  // Use only the specified fraction of the training data in the do_target_task_training phase.
+                        // should not impact pretrain training, even for the same task.
+                        // training_data_fraction should not be used together with                 // target_train_data_fractio
```

Comment:
"fraction"

Also, random request: can you add that it's a random fraction of the data (I think)?

Reply:
It seems to always be set to 1.0